### PR TITLE
Document rotations

### PR DESCRIPTION
**Pull request by ColinRCox**
This is all the rotations information.

### CODE REVIEW

File: pysiaf/tests/test_rotations.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+dec2 = 69.0
+
+
+def test_attitude(verbose=False):
+    """ Tests the properties of the attitude matrix in calculating positions and roll angles. Incidentally tests underlying
+    matrix generation and vector construction  and conversions to Euler angles
+
+    Parameters
+    ----------
+    verbose     logical - set to true if detailed print-out needed
```
Comment (Johannes-Sahlmann) [inline pysiaf/tests/test_rotations.py]: Please adhere to numpy docstring formatting, i.e. add the argument type after `:` and move the description to a new line. See https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes

File: pysiaf/utils/rotations.py [hunk 2]
```diff
@@ -12,17 +12,30 @@
...
 
 def attitude(v2, v3, ra, dec, pa):
     """This will make a 3D rotation matrix which rotates a unit vector representing a v2,v3 position
     to a unit vector representing an RA, Dec pointing with an assigned position angle
-    Described in JWST-STScI-001550, SM-12, section 6.1"""
+    Described in JWST-STScI-001550, SM-12, section 6.1
+
+    Parameters
+    ----------
+    v2  a position measured in arc-seconds
```
Comment (Johannes-Sahlmann) [inline pysiaf/utils/rotations.py]: Convert to numpy docstring format.

File: pysiaf/utils/rotations.py [hunk 3]
```diff
@@ -43,36 +56,62 @@ def attitude(v2, v3, ra, dec, pa):
 
 
 def axial(ax, phi, u):
```
Comment (Johannes-Sahlmann) [inline pysiaf/utils/rotations.py]: I suggest renaming this function to `axial_rotation`

File: pysiaf/utils/rotations.py [hunk 4]
```diff
@@ -12,17 +12,30 @@
 
 """
 from __future__ import absolute_import, print_function, division
-
-# from math import *
 import numpy as np
+from math import sqrt, sin, cos, atan2, asin, acos, radians, degrees
```
[the remark this answers is not in the document]
Comment (ColinRCox) [reply pysiaf/utils/rotations.py]: I have now used the numpy functions where arrays are to be addressed

File: pysiaf/tests/test_rotations.py (hunk 1 above)
Comment (ColinRCox) [reply pysiaf/tests/test_rotations.py]: I have attended to this

File: pysiaf/utils/rotations.py (hunk 2 above)
Comment (ColinRCox) [reply pysiaf/utils/rotations.py]: Done

**Pull request merged**